client/my-sites/jetpack-search/upsell.tsx

```diff
@@ -22,10 +22,7 @@ export default function JetpackSearchUpsell(): ReactElement {
 	const onUpgradeClick = useTrackCallback( undefined, 'calypso_jetpack_search_upsell' );
 	const siteSlug = useSelector( getSelectedSiteSlug );
 	const translate = useTranslate();
-	const upgradeUrl =
-		'/checkout/' +
-		siteSlug +
-		'/jetpack_search_monthly?utm_campaign=my-sites-jetpack-search&utm_source=calypso';
+	const upgradeUrl = 'https://jetpack.com/upgrade/search/?site=' + siteSlug;
```

Comment:
Can we keep the `utm_campaign` and `utm_source` in the query string for Google Analytics? We used to have them previously when we linked to upgrade/search: https://github.com/Automattic/wp-calypso/pull/49346/files